src/absil/illib.fs

```diff
@@ -14,7 +14,7 @@ open Internal.Utilities.Collections
 // Code that uses this should probably be adjusted to use unsigned integer types.
 let (>>>&) (x:int32) (n:int32) = int32 (uint32 x >>> n)
 
-let notlazy v = Lazy.CreateFromValue v
+let notlazy v = Lazy<_>.CreateFromValue v
```

Comment:
Yes it no longer compiles
````
<snip>
    C:\KevinRansom\visualfsharp\src\absil\illib.fs(17,17): error FS1124: Multiple types exist
    called 'Lazy', taking different numbers of generic parameters. Provide a type instantiation
    to disambiguate the type resolution, 
   e.g. 'Lazy<_>'.
   [C:\KevinRansom\visualfsharp\src\fsharp\FSharp.Compiler\FSharp.Compiler.fsproj]
</snip
````
**mscorlib.dll** only has:
* **System.Lazy<T>** whereas 

**System.Runtime.dll** has:
* **System.Lazy<T>** and 
* **System.Lazy<T,M>**

In the desktop framework:
* **System.Lazy<T,M>** is in **System.ComponentModel.dll**